index.js

```diff
@@ -332,15 +299,21 @@ class LatticeKeyring extends EventEmitter {
         if (!this.isUnlocked())
           return reject(new Error('No connection to Lattice. Could not send request.'));
         this.sdkSession.sign(req, (err, res) => {
-          if (err)
+          if (err) {
             return reject(new Error(err));
-          if (!res.sig)
+          }
+          if (!this._syncCurrentWalletUID()) {
```

Comment:
Should return `activeWallet.uid | undefined`, rather than `(true|false)`. The call-site doesn't need to change, and it makes the internal side-effect more explicit.

Reply:
You're saying to make it not fail? We don't use the response except to throw an error.

Reply:
It can fail; simply put: the function returns the active UID, or `undefined`. The latter triggers the `reject`.

Reply:
Okay, but we never use active UID returned from this. But that's fine

Reply:
Updated in force push: [d07f855](https://github.com/GridPlus/eth-lattice-keyring/pull/28/commits/d07f855d84570001c40d04759500a95adb87046b)